outputs.tf

```
@@ -58,14 +58,14 @@ output "efs_mount_targets" {
   description = "The mount targets for the EFS file share."
 }
 
-output "email_sending_domain_certreadrole" {
+output "email_sending_domain_certreadroles" {
   value       = module.email_sending_domain_certreadrole
-  description = "The IAM role that allows for reading the certificate for the email-sending domain."
+  description = "The IAM roles that allow for reading the certificate for the email-sending domain."
```

Just mirrors the README suggestion for convenience.

```suggestion
  description = "The IAM roles that allow for reading the certificate for each email-sending domain."
```